src/foam/u2/table/TableHelperMixin.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.table',
+  name: 'TableHelperMixin',
```

Comment:
Does this need to be a mixin? Both uses extend View, so it could be a TableComponentView that they both extend.